modules/cudacodec/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ ocv_glob_module_sources()
 
 set(extra_libs "")
 
+if(HAVE_FFMPEG)
```

Comment:
Perhaps this should not work with `HAVE_FFMPEG_WRAPPER` (Windows)

Reply:
Would it be possible to quickly explain the difference between HAVE_FFMPEG and HAVE_FFMPEG_WRAPPER